Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -23,8 +23,8 @@
 
     <!-- XTXT: Text bodies for the release info screen bullet points -->
     <string-array name="new_release_body">
-        <item>Als Organisator eines Events können Sie nun Ihre Gäste in Vertretung für eine positiv getestete Person warnen, die am Event teilgenommen hat, aber nicht über die Corona-Warn-App eingecheckt war.</item>
-        <item>Für Genesene wird ab jetzt direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei fast allen Impfstoffen.</item>
+        <item>Als Organisator eines Events können Sie nun nach Aufforderung durch das Gesundheitsamt Ihre Gäste in Vertretung für eine positiv getestete Person warnen, die am Event teilgenommen hat, aber nicht über die Corona-Warn-App eingecheckt war.</item>
+        <item>Für Genesene wird ab jetzt direkt nach der ersten Impfung der vollständige Impfschutz in der Corona-Warn-App angezeigt. Die zweiwöchige Wartezeit entfällt bei allen Impfstoffen außer bei Johnson &amp; Johnson.</item>
```

Comment:
@mtwalli, `&amp` works too but should we surroung the text with double quotes instead to have consistency with the rest of the strings?

Reply:
`""` did not fix it for me  , AS still complains about it that it should be escaped